tests/jballerina-unit-test/src/test/resources/test-src/documentation/deprecated_annotation_project/main.bal

```diff
@@ -93,6 +93,17 @@ public type Bank record {|
     CountryCode countryCode;
 |};
 
+# Employee record
+#
+# + name - street Parameter Description
+# + age - city Parameter Description
+# # Deprecated
```

Comment:
Is this allowed now? If so, this doesn't seem correct to me. Isn't this dangling deprecated documentation? Can't we use deprecated docs at field level?

cc @KavinduZoysa. Was this discussed when doing the object field deprecation changes?

Reply:
I am not sure whether we have discussed this. @prakanth97 can you please clarify this in spec?

Reply:
This example gave `invalid documentation: 'Deprecated' documentation is only allowed on constructs annotated as '@deprecated'`. It should be corrected to 
```
# Employee record
#
# + age - city Parameter Description
public type Employee record {|
    # # Deprecated
    @deprecated
    string name;
    int age;
|};
```